### PR TITLE
Add logo file and change reference to new logo

### PR DESCRIPTION
**Pull request by david4096**
This PR adds an SVG logo created by Kim and replaces the Dockstore logo in the upper left with a logo with text:

image

### CODE REVIEW

**Comment by david4096** [comment]
Also changes the navbar entry from "Help" to "Forum" following Kim's suggestions

image

**Review by GFJHogue — APPROVED** [review]
Looks good!

**Review by NatalieEO — APPROVED** [review]

**Review by Richard-Hansen — APPROVED** [review]
The new logo looks great!

File: src/app/navbar/navbar.component.html [hunk 1]
```diff
@@ -30,7 +30,7 @@
           <img class="site-icons-small hidden-sm" src="../assets/images/dockstore/dockstore-documentation.png" alt="stacked pages" /> Docs
         </a>
         <a mat-button class="icon" [href]="Dockstore.DISCOURSE_URL" target="_blank" rel="noopener noreferrer">
-          <mat-icon class="hidden-sm">help_outline</mat-icon> Help
+          <mat-icon class="hidden-sm">forum</mat-icon> Forum
```
Comment (coverbeck) [inline src/app/navbar/navbar.component.html:33]: What is this change? I don't see the difference in the screenshot.

**Review by denis-yuen — APPROVED** [review]

File: src/app/navbar/navbar.component.html (hunk 1 above)
Comment (coverbeck) [reply src/app/navbar/navbar.component.html:33]: > What is this change? I don't see the difference in the screenshot.

Ah, I had only looked at the first screen shot, in the PR description. Now I see it.

**Review by coverbeck — APPROVED** [review]

**Review by garyluu — APPROVED** [review]

**Pull request merged**